source/static-site/pages/pages.js

```diff
@@ -1,5 +1,5 @@
 // NOTE: Do not edit this file. It is built by running `./scripts/create-static-site-files.js`
```

Comment:
should generated files really be added to Git?

Reply:
added to gitignore